ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -655,6 +679,7 @@ open class DrawerController: UIViewController {
     private var resizingHandleIsInteractive: Bool {
         return resizingBehavior == .dismissOrExpand
     }
+    private let screenBound: CGFloat = UIScreen.main.bounds.height
```

Comment:
This seems to be used only in one method. It should be a variable local to that scope.
Also, getting the screen bounds with UIScreen is outdated. It should be retrieved from the UIView's window property instead.

Reply:
changed this.